pyproject.toml

```diff
@@ -21,3 +21,86 @@ extension-pkg-whitelist = [
     "pydantic"
 ]
 load-plugins = "pylint_pytest"
+
+[tool.ruff]
+line-length = 88
+target-version = "py38"
+src = ["starcraft", "tests"]
```

Comment:
```suggestion
src = ["rockcraft", "tests"]
```

and then the Makefiles targets should need these dirs right?

Reply:
Ugh yeah I always miss one. Thank you!

W.r.t. the Makefile target, it still needs these since they're doing different things. This says that "rockcraft" and "tests" are first-party code, whereas the command-line arguments say to check those directories. Not the clearest...